document/js-api/index.bs

```diff
@@ -886,18 +886,21 @@ Note: [=call an Exported Function|Calling an Exported Function=] executes in the
 Note: Exported Functions do not have a \[[Construct]] method and thus it is not possible to call one with the `new` operator.
 
 <div algorithm>
-  To <dfn>create a host function</dfn> from the JavaScript object |func|, perform the following steps:
+  To <dfn>create a host function</dfn> from the JavaScript object |func| and type |functype|, perform the following steps:
 
     1. Let |hostfunc| be a [=host function=] which performs the following steps when called:
-        1. If the signature contains an [=𝗂𝟨𝟦=] (as argument or result), the host function throws a {{TypeError}} when called.
-        1. Let |arguments| be a [=list=] of the arguments of the invocation of this function.
-        1. Let |jsArguments| be an empty [=list=].
-        1. For each |arg| in |arguments|,
-            1. [=Append=] [=ToJSValue=](|arg|) to |jsArguments|.
-        1. Let |ret| be ? [=Call=](|func|, undefined, |jsArguments|). If an exception is thrown, trigger a WebAssembly trap, and propagate the exception to the enclosing JavaScript.
-        1. Let [<var ignore>parameters</var>] → [|results|] be |functype|.
-        1. If |results| is empty, return undefined.
-        1. Otherwise, return [=ToWebAssemblyValue=](|ret|, |results|[0]).
+        1. Try running the following steps:
```

Comment:
I didn't know you were allowed do this in spec-language. Is there another example you can reference with this pattern?

Reply:
I copied the phrasing from https://heycam.github.io/webidl/#dfn-attribute-getter

Reply:
It isn't supported by the [Algorithmic Conventions](https://tc39.github.io/ecma262/#sec-algorithm-conventions) of the ES spec, which requires exceptional results to be handled explicitly through completion values and the use of [auxiliary meta operators and conventions](https://tc39.github.io/ecma262/#sec-runtime-semantics) like `?`. It seems like the WebIDL prose does not follow the ES conventions, while the Wasm API tries to, as witnessed by its use of `?`. Is there a strong reason to deviate from these conventions in this case?

Reply:
This specification is written with Web-style conventions with a couple ES-isms mixed in where needed, like WebIDL.